docker-compose.yml

```diff
@@ -1,50 +1,42 @@
-version: '3'
+version: '3.9'
 
 services:
     
   db:
     image: postgres
     volumes:
       - ./tmp/db:/var/lib/postgresql/data
+      - ./db/init.sql:/docker-entrypoint-initdb.d/init.sql
     restart: always
-    env_file:
-      - .env
     environment:
-      POSTGRES_USER: ${DB_USERNAME}
-      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: password
     ports:
       - 5432:5432
-    expose:
-      - "5432"
 
   api:
     build:
       context: .
-      dockerfile: Docker_rails
+      dockerfile: Dockerfile
     volumes:
       - .:/app
```

Comment:
Since `.` includes the `client` which gets its own container, there's no need to mount the entire directory here. The ideal solution is to specify each directory you want to mount in `x-vars` and then reference the volumes here. 

Like this:

https://github.com/puipuipartpicker/kbpartpicker/blob/f787c63227b16e480bc1b49f91af897e144ac248/docker-compose.yml#L3-L17